docs/source/fft.rst

```diff
@@ -0,0 +1,21 @@
+.. role:: hidden
+    :class: hidden-section
+
+torch.fft
+===================================
+
+Discrete Fourier transforms and related functions.
+
+To use these functions the torch.fft module must be imported since its name
+conflicts with the :func:`torch.fft` function.
```

Comment:
nit: I'd say something like the deprecated, soon to be deprecated, etc.

Reply:
It'll have to be in a follow-up since the torch.fft functionality is not the same as numpy.fft.fft's, so we'll need a more thorough deprecation plan.